pkg/vulnsrctest/get.go

```diff
@@ -0,0 +1,31 @@
+package vulnsrctest
```

Comment:
my suggestion for naming:
folder: 'vulnsrc'
package: 'vulnsrc_test'
filename: 'get_test.go'

Reply:
We can't use package: `vulnsrc_test` or filename: `get_test.go`.
Golang thinks this is a test package and we can't import functions from a test package.